mqtt3/1.0.0/modules/ROOT/pages/mqtt3-connector-listener.adoc

```diff
@@ -0,0 +1,121 @@
+= Listen for New Messages with MQTT3 Connector
+
+The Anypoint Connector for MQTT3 (MQTT3 Connector) *On New Message* (`<mqtt3:listener>`) source enables you to listen for new incoming messages using one or more topic filters (`topicFilter`). Each topic filter has a specific quality of service (`qos`) configured.
+
+In the Configuration XML editor, the `<mqtt3:listener>` and `<mqtt3:topic>` configurations look like this:
```

Comment:
Maybe indicate that this is an example, either in the section heading or in this sentence: "In this example, the..."